website/docs/r/virtual_hub_ip_configuration.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_virtual_hub_ip_configuration"
+description: |-
+  Manages a Virtual Hub IP Configuration.
+---
+
+# azurerm_virtual_hub_ip_configuration
+
+Manages a Virtual Hub IP Configuration.
+
+~> **NOTE** Virtual Hub IP Configuration only supports Standard Virtual Hub without Virtual Wan.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_hub" "example" {
+  name                = "example-vhub"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  sku                 = "Standard"
+}
+
+resource "azurerm_public_ip" "example" {
+  name                = "example-pip"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  allocation_method   = "Dynamic"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-vnet"
+  address_space       = ["10.5.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefix       = "10.5.1.0/24"
+}
+
+resource "azurerm_virtual_hub_ip_configuration" "example" {
+  name                         = "example-vhubipconfig"
+  virtual_hub_id               = azurerm_virtual_hub.example.id
+  private_ip_address           = "10.5.1.18"
+  private_ip_allocation_method = "Static"
+  public_ip_address_id         = azurerm_public_ip.example.id
+  subnet_id                    = azurerm_subnet.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Virtual Hub IP Configuration. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Virtual Hub IP Configuration should exist. Changing this forces a new resource to be created.
```

Comment:
can this be different then the RG of the virtual hub? the id seemsto be based off the hub indicating we can grab this from the virtual hub id

Reply:
updated